conda/recipes/cudf_kafka/meta.yaml

```diff
@@ -41,6 +41,9 @@ requirements:
     - {{ compiler('cxx') }}
     - ninja
     - sysroot_{{ target_platform }} {{ sysroot_version }}
+    {% if cuda_major == "12" %}
+    - cuda-gdb
```

Comment:
Just leaving a comment to document that this is an issue caused by https://github.com/rapidsai/ci-imgs/pull/77. We removed the system CTK from CUDA 12 ci-conda images, and cudf_kafka's build script relied on a system `cuda-gdb` being found. That was a bug (!) exposed by my changes to the CI images. This should be removed in #14292 once CI is unbroken and we rewrite cudf_kafka's build system to use scikit-build.